tools/UpdateVendors/Datadog.Dependabot.Honeypot.txt

```diff
@@ -0,0 +1,11 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
Can we add the word `template` to this file? Maybe instead of the `txt` extension?

Reply:
Done